rust/src/nasl/interpreter/error.rs

```diff
@@ -18,12 +19,12 @@ pub struct FunctionError {
     pub function: String,
     /// Kind of error
     #[source]
-    pub kind: FunctionErrorKind,
+    pub kind: NaslError,
```

Comment:
Nit: I am not sure if I like the term  NaslError as it is just for builtin functions.

Reply:
That makes sense. I guess I'll rename it back to `FunctionErrorKind` for this draft (to make the changes I made more apparent and not flood it with 40 files of rename changes) and then we can discuss the name at some later point.